src/components/app/details/cicdHistory/History.components.tsx

```diff
@@ -102,6 +113,33 @@ export const GitChanges = ({ gitTriggers, ciMaterials }: GitChangesType) => {
                     </div>
                 ) : null
             })}
+            {artifact && userApprovalMetadata && (
```

Comment:
same here

Reply:
I think we can keep this as internal part which was specific has been moved